lib/utils/local-process.js

```diff
@@ -60,7 +60,7 @@ Please ensure the content folder has correct permissions and try again.`));
             cp.on('message', (msg) => {
                 if (msg.error) {
```

Comment:
this would need to be updated to be `msg.message` I think.

Reply:
Why? We forward `error: true`.

Reply:
ah, sorry. Misread a couple of things 🤦‍♂️

Reply:
No problem :)